docs/content/guides/dapp-deployment/mainnet-deployment.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Mainnet Deployment Guidelines
+sidebar_title: 4. Deploy to Mainnet
+---
+
+In the early days of the Flow network, applications deployed to the Flow Mainnet are required to be review and tested by the Flow team. For details about the testing and approval process, please read through our guide \[Testnet Testing Guidelines](../testnet-testing) If your app has passed the review process you're ready to deploy on Flow's Mainnet!
```

Comment:
This link isn't showing up correctly right now on the preview site, i assume that'll change once it's deployed?

Reply:
It's incorrect.